packages/cli/src/databases/entities/abstract-entity.ts

```diff
@@ -23,6 +23,8 @@ const timestampSyntax = {
 
 export const jsonColumnType = dbType === 'sqlite' ? 'simple-json' : 'json';
 export const datetimeColumnType = dbType === 'postgresdb' ? 'timestamptz' : 'datetime';
+export const datetimeColumnDefault =
```

Comment:
let's drop this, and if we really want to optimize the storage for timestamps on sqlite, we can do that for all timestamp columns in the system in a single PR.
That way we don't have to have 2 different types of timestamps in the app.